BritneyS/portfolio-site-gatsby
# Project Section

## What does this PR do :question:
This PR adds the Project section to the portfolio page, using `Card` components from Material Kit. Those components were added to reusable components that are populated by the provided `projects` data.


## Any packages added :package:
N/A


## How to test it :microscope:
1. Checkout this branch and run `gatsby develop` to start the project.
1. Navigate to `https://localhost:8000`
1. **Expected:** Confirm that the project section appears, between two separators, where two project cards are rendered. The top card should have a Github link and a demo link, and the other card should just have a Github link.
1. **Expected:** Scroll to the top of the screen, and in the header, click on "Projects". Confirm that the page scrolls to the Projects section.
1. **Expected:** Confirm that the Github link in the first project card links to the repo for this project (`https://github.com/BritneyS/portfolio-site-gatsby`), and the demo link navigates to the portfolio site.
1. **Expected:** Confirm that the Github link in the second project card links to the user page for this Github account (`https://github.com/BritneyS`).


## Any background info you would like to include :white_check_mark:



## Screenshots (if applicable) :camera:
<img width="963" alt="Screen Shot 2020-03-08 at 11 50 01 AM" src="https://user-images.githubusercontent.com/8409475/76166184-fc74e600-6132-11ea-9fb5-1dda46c9d42d.png">

<img width="1121" alt="Screen Shot 2020-03-08 at 11 35 49 AM" src="https://user-images.githubusercontent.com/8409475/76165893-03025e00-6131-11ea-96cb-fd7739ea12a2.png">

![Mar-08-2020 11-39-21](https://user-images.githubusercontent.com/8409475/76165980-98055700-6131-11ea-941f-b9ff6c9dd472.gif)
